paxml/main.py

```diff
@@ -360,13 +380,22 @@ def run(experiment_config: base_experiment.BaseExperiment,
 
 
 def main(argv: Sequence[str]) -> None:
+  start = time.time()
   if len(argv) > 1:
     raise app.UsageError('Too many command-line arguments.')
 
+  if FLAGS.tfds_data_dir is not None:
+    seqio.set_tfds_data_dir_override(FLAGS.tfds_data_dir)
```

Comment:
@ashors1 can you move the seqio here

  if FLAGS.tfds_data_dir is not None:
    # seqio import is slow so avoid module level 
    import seqio
    seqio.set_tfds_data_dir_override(FLAGS.tfds_data_dir)